CMakeLists.txt

```diff
@@ -1,6 +1,16 @@
-cmake_minimum_required (VERSION 2.8)
+cmake_minimum_required (VERSION 3.14)
```

Comment:
This should build and install a paragon library, plus the headers; but probably then doesn't need to install the driver as most people will never use it. 

Once we pull this into Zeek via submodule, it'll need some more tweaking to fit into its CMake setup.